src/sinks/hdfs/mod.rs

```diff
@@ -0,0 +1,75 @@
+//! `hdfs` sink.
+//!
+//! This sink will send it's output to hdfs.
+//!
+//! `hdfs` is an opendal based services. This mod itself only provide
```

Comment:
```suggestion
//! `hdfs` is an OpenDAL based service. This mod itself only provide
```

**nit-picking (not blocking)**: this seems to be the proper casing for the name.